test/auth_section/sign_in/widget_test.dart

```diff
@@ -27,6 +27,28 @@ class MockFirebaseAuthService extends AuthService {
   @override
   Stream<UserData> get streamOfAuthState {}
 
+  @override
```

Comment:
It's not really part of this PR but for clarity I think the `MockFirebaseAuthService` class should be `FakeAuthService` and `implement` rather than `extend` AuthService. You will get the same result but it will be clearer what the intention is, IMO.

Reply:
just read the mockito fake documentation, thanks for the suggestion! its alot cleaner only rewriting the methods im using for the test.